### PR TITLE
Engineering Fuel Cans Now Contain Sulphuric Acid

### PR DESCRIPTION
**Pull request by DorainGrey**
## What this does
Engineering Fuel cans come stocked with Sulfuric acid in their 2nd slot, alongside the welding fuel. 
## Why it's good
Fuelcans are a nice feature but hardly used at all, also it just makes sense to use the cans instead of bottles. It's not like acid is rare.

## Changelog

:cl:
 * tweak: Engineer Fuelcans now come stocked with Sulphuric Acid.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Not sure that I'm a fan of this, mostly because it reduces your choice. You have to choose to dump something out (e.g.: splatter the floor in welding fuel or waste the limited engineering stock of acid) if you want to carry something else. Do most people use fuel cans for fuel+acid? Absolutely. But the option is always there to carry a different engineering-useful reagent, like water to quick-fill your foam extinguisher, ethanol for the TEG, or an antitox mix for walking past the radioactive shit in the department.

In short: it has never felt like a big burden to me to click the acid into the fuel can at the start of shift.

**Comment by Eneocho** [comment]
Clearly the solution to kurf's issue is to add a third empty storage slot where you can pour whatever you want.

**Comment by DorainGrey** [comment]
> Clearly the solution to kurf's issue is to add a third empty storage slot where you can pour whatever you want.

I'd do this tbh, that address your concern @Kurfursten ?

**Comment by Kurfursten** [comment]
I don't strongly object to the idea, although you will need a new sprite and there is a little bit of worry that keeping track of your reagents in each slot will become less intuitive. It takes a lot less passive brain power to remember "this is the one I have active" than "I need to click twice to toggle to slot three to access my water, then click one more time to get back to fuel".

Maybe I'm underestimating engineers, but the point of the can is of course to reduce your need for thinking about things other than engines, not increase it.

**Comment by Eneocho** [comment]
>It takes a lot less passive brain power to remember "this is the one I have active" than "I need to click twice to toggle to slot three to access my water, then click one more time to get back to fuel".

A toggle could work for that. If disabled you can access either the fuel or the acid. If enabled you can only access the extra container. That way the "use to swap to acid/fuel" experience is maintained unless you actively want to use the extra slot.

**Comment by DorainGrey** [comment]
> > It takes a lot less passive brain power to remember "this is the one I have active" than "I need to click twice to toggle to slot three to access my water, then click one more time to get back to fuel".
> 
> A toggle could work for that. If disabled you can access either the fuel or the acid. If enabled you can only access the extra container. That way the "use to swap to acid/fuel" experience is maintained unless you actively want to use the extra slot.

Honestly, thinking about it for a sec, it's probably easier and better for everyone if I just replace the welding fuel with sulfuric, since it's easy to get welding fuel. I can see what Kurf means about the third slot, doesn't really fit.



Edit: Going to leave as is, since it seems the most popular option.

**Comment by calzilla1** [comment]
Maybe some sort of bisected fuel can would help with this. One item with two reservoirs, one side for fuel and the other for acid, and alt clicking would change which side it dispenses (or just automate it, not like welding fuel and acid share any tools)

**Comment by Kurfursten** [comment]
Is that a joke or are you actually unaware that is the item we are discussing right now, which works as you describe (toggle by use inhand, not alt click)

**Comment by Eneocho** [comment]
>Maybe some sort of bisected fuel can would help with this. One item with two reservoirs, one side for fuel and the other for acid, and alt clicking would change which side it dispenses (or just automate it, not like welding fuel and acid share any tools)

That's how it works already. The can is bisected and has fuel on one slot and is empty on the other, with the idea being that you fill that other side with acid, water, ethanol, booze, etc.
Kurf's issue is that this PR makes it so it starts with fuel on one side and acid on the other, which makes it so you have to pour one of those out (either wasting the acid or leaving fuel puddles). I suggested adding a 3rd slot that's empty, so you can have both liquids and an extra storage slot, but Dorain decided to just leave them as is.

**Comment by Eneocho** [comment]
Instead of removing the sacid bottles, move them to the contraband list.

**Comment by pedr69** [comment]
>move to contraband

why not have opening the glubb closets take 5 seconds too? why not gate every engineering tool behind a wait timer
for what purpose

**Comment by jwhitak** [comment]
I don't like the removal of the dedicated sulfuric acid bottles and I don't like acid placed into the spare fuel tank of the fuel cans by default.
I can see why engies who like using both the welder and the soldering iron would want it in one item, but just putting it into the fuel can by default and still calling it a fuel can is lazy. Spice it up at the very least. Call it something new like, Engineering Fluid Filler-Upper. EFF-U for short. That way people who don't read the PRs or changelogs might know where to look for sulfuric acid.

**Comment by Eneocho** [comment]
> > move to contraband
> 
> why not have opening the glubb closets take 5 seconds too? why not gate every engineering tool behind a wait timer for what purpose

I see your point but consider this: This PR is making the cans have both fuel and sacid. Thus no sense in having the regular sacid bottles on the normal list. Moving it to contraband keeps the extra sacid bottles and gives a reason to hack the thing if you desperately need more sacid (since 3 potato batteries and a weak foam 'nade aren't really an incentive). In effect it'd be a buff to the engivend. You not only get your 250u Sacid via fuel cans (50x5), but you also get the extra 90u via the existing vials (30x3).

**Comment by Eneocho** [comment]
The cans could change their general icon depending on the side that's currently open. The fuel one is orange while the sacid one is white, for example. Would give you a clear visual indicator as to which side is currently active (the slider on the top moves but that's not that obvious).

**Comment by Kurfursten** [comment]
That would be more confusing, because fuel is dark red and sacid is orange. And what if you put different reagents in?

**Comment by Eneocho** [comment]
> That would be more confusing, because fuel is dark red and sacid is orange. And what if you put different reagents in?

The idea is just having 2 visually distinct sprites so you can check at a glance.

**Comment by Dacendeth** [comment]
Just dont remove the small acid bottles and the majority of those thumbs downs will go away.

**Comment by DorainGrey** [comment]
> Just dont remove the small acid bottles and the majority of those thumbs downs will go away.

Done, change your vote.

**Comment by Kurfursten** [comment]
Gotta make a call on this, it's 7 days. Tough one. Surprised it's this close.

Maybe a little creepy, reducing the need to ever visit chemistry (or ask mechanics/science) (or ask cargo for a tank) for a refill (which is in a way the point of the solder in its original incarnation) since there is now an additional 150 acid. That's a little sad to lose, especially because 90 acid will last most shifts. 

Tools get added to increase interaction, people grow reliant on them, and they become necessities. It is ruled unreasonable to interact. This is the way of things, maybe.
